package.json

```diff
@@ -93,6 +96,7 @@
     "moment": "^2.13.0",
     "morgan": "^1.7.0",
     "node-sass": "^3.6.0",
+    "nsp": "^2.4.0",
```

Comment:
Are we running this on Travis and deployment?  I am worried that we are installing a CLI that likely needs to be a local binary that will slow down things.